document/core/text/modules.rst

```diff
@@ -359,7 +359,11 @@ Abbreviations
 
 Multiple imports with the same ${:nm_1} may be declared together:
 
-$${grammar: Timports_/abbrev}
+$${grammar: Timports_/abbrev-compact1}
+
+Multiple imports with the same ${:nm_1} and ${:xt} may also be declared together, in which case identifiers may be placed on individual items instead of the ${:externtype}:
```

Comment:
```suggestion
Multiple imports with the same name ${:nm_1} and :ref:`external type <syntax-externtype>` ${:xt} may also be declared together, in which case identifiers may be placed on individual items instead of the external type:
```